stm/stm32f4xx/stm32f4xx_hal_init.c

```diff
@@ -4,7 +4,7 @@
  * @brief This function is executed in case of error occurrence.
  *
  */
-static void Error_Handler(void) {
+__attribute__((weak)) void Error_Handler(void) {
```

Comment:
The STM driver code uses __weak also I used it in ADC ;\ . Maybe change to `__weak` for uniformity?